src/internal/components/chart-series-details/styles.scss

```diff
@@ -55,8 +55,10 @@ $font-weight-bold: awsui.$font-weight-heading-s;
 
 .list {
   display: flex;
-  margin: 0;
-  padding: 0;
+  margin-block: 0;
+  margin-inline: 0;
+  padding-block: 0;
+  padding-inline: 0;
   list-style: none;
   flex-direction: column;
 
```

Comment:
this file also has margin-top
https://github.com/cloudscape-design/components/blob/a011f57bf4c17a6ae236449b9562093df20a19ab/src/internal/components/chart-series-details/styles.scss#L78

and width 
https://github.com/cloudscape-design/components/blob/a011f57bf4c17a6ae236449b9562093df20a19ab/src/internal/components/chart-series-details/styles.scss#L89

Reply:
Good find! I wonder why those weren't flagged by the linter. Investigating...